features/tasks.feature

```diff
@@ -261,6 +261,54 @@ Feature: Tasks
       }
       """
 
+  Scenario: Add task to a group
+    Given the fixtures files are loaded:
+      | sylius_channels.yml |
+      | tasks.yml           |
+      | users.yml           |
+    And the user "bob" has role "ROLE_ADMIN"
+    And the user "bob" is authenticated
+    And the tasks with comments matching "#bob" are assigned to "bob"
+    When I add "Content-Type" header equal to "application/ld+json"
+    And I add "Accept" header equal to "application/ld+json"
+    And the user "bob" sends a "PUT" request to "/api/tasks/1/add_to_group/1" with body:
```

Comment:
IMHO it would be more restful to do it like this:

```
POST /api/task_groups/1/tasks

{
  "tasks":["/api/tasks/1"]
}
```

Reply:
If we add the operation at the group level, we can add multiple tasks to a group with just one request, no?

```
POST /api/task_groups/1/tasks

{
  "tasks":[
    "/api/tasks/1",
    "/api/tasks/2",
    "/api/tasks/3"
  ]
}
```

Reply:
Yes, I agree. I'll try to make this :)